app/src/main/java/com/example/trainingintive/dog_images_feature/data/repository/DogImageRepository.kt

```diff
@@ -3,21 +3,22 @@ package com.example.trainingintive.dog_images_feature.data.repository
 import com.example.trainingintive.dog_images_feature.data.local.DogImageDao
 import com.example.trainingintive.dog_images_feature.data.network.DogImageApiService
 import com.example.trainingintive.dog_images_feature.domain.model.DogImageUrl
-import io.reactivex.rxjava3.core.Completable
-import io.reactivex.rxjava3.core.Flowable
-import io.reactivex.rxjava3.core.Single
 import javax.inject.Inject
 
 class DogImageRepository @Inject constructor(
     private val dogImageApiService: DogImageApiService,
     private val dogImageDao: DogImageDao
 ) {
 
-    fun getDogImageUrl(): Single<DogImageUrl> =
+    fun getDogImageUrl() =
         dogImageApiService.getDogImage()
             .map { it.toDomain() }
 
-    fun getAllImageUrlsFromLocalDatabase(): Flowable<List<DogImageUrl>> = dogImageDao.getAllImageUrls()
+    fun getAllImageUrlsFromLocalDatabase() = dogImageDao.getAllImageUrls()
```

Comment:
For the app and domain layers, it isn't important HOW you store saved image URLs. So it will be better to name such functions like `getAllStoredImageUrls` or something like this

Reply:
And same for other functions